src/components/Swiper/DefaultNavigation/index.tsx

```diff
@@ -53,17 +50,15 @@ const navigationDirectionStyle: { [key in NavigationDirection]: FlattenSimpleInt
   `,
 };
 
-const Button = styled(IconButton)<{ direction: NavigationDirection }>`
+const Button = styled(IconButton)<{ direction: string }>`
```

Comment:
```suggestion
const Button = styled(IconButton)<{ direction: keyof navigationDirectionStyle }>`
```

이렇게 해주면 direction 에 right, left 외에 값을 잘못 넣어 생기는 에러를 방지할 수 있습니다.
하지만 이렇게 타이핑을 하면, Button의 코드가 navigationDirectionStyle의 값에 종속되게 됩니다.

이 종속성을 역전시키기 위해서는 기존 코드의 `NavigationDirection` 을 다시 살리는게 좋겠습니다.

`NavigationDirection` 를 살려주세요